JPSimulatorHacksSampleTests/JPSimulatorHacksSampleTests.m

```diff
@@ -36,6 +36,10 @@
 #import <KIF/UIAccessibilityElement-KIFAdditions.h>
 #import <XCTest/XCTest.h>
 
+#ifdef IS_IOS9_AND_UP
```

Comment:
Hi, I will check this as soon as possible and let you will know.

> On 09 Aug 2016, at 08:21, Johannes Plunien <notifications@github.com> wrote:
> 
> In JPSimulatorHacksSampleTests/JPSimulatorHacksSampleTests.m:
> 
> > @@ -36,6 +36,10 @@
> >  #import <KIF/UIAccessibilityElement-KIFAdditions.h>
> >  #import <XCTest/XCTest.h>
> >  
> > +#ifdef IS_IOS9_AND_UP
> This is always true. Probably the reason for the failing tests?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>